example/burrow_integration/integ-ex.py

```diff
@@ -0,0 +1,166 @@
+    # bytecode was generated using remix editor  https://remix.ethereum.org/
+    tx = iroha.transaction([
+        iroha.command('CallEngine', caller='admin@test', input=bytecode)
+    ])
+    IrohaCrypto.sign_transaction(tx, ADMIN_PRIVATE_KEY)
+    net.send_tx(tx)
+    hex_hash = binascii.hexlify(IrohaCrypto.hash(tx))
+    print(hex_hash)
+    for status in net.tx_status_stream(tx):
+        print(status)
+    return hex_hash
+        
+def get_eng_rec(hash):
+    query = iroha.query('GetEngineReceipts', tx_hash=hash)
+    IrohaCrypto.sign_query(query, ADMIN_PRIVATE_KEY)
+
+    response = net.send_query(query)
+    print(response)
+    contract_add = response.engine_receipts_response.engine_receipts [0].contract_address
+    print (contract_add)
+    return contract_add
+
+@trace
+def addasset(address) :
+    params = ("37410dfa"   # the first 4 bytes of the keccak256 hash of the function signature:‘addAsset(string,string)’
+            "0000000000000000000000000000000000000000000000000000000000000040" # the location of the data part of the first parameter
+            "0000000000000000000000000000000000000000000000000000000000000080" # the location of the data part of the second parameter
+            "000000000000000000000000000000000000000000000000000000000000000b" # count for "coin#domain"
```

Comment:
it would be much better to have function which would generate those hashes

Reply:
I mean something like that:
```
def make_number_left_padded(number: int, width: int=64):
    return str(number).zfill(width)
```
it will from `make_number_hex_left_padded(5)` generate:
```
0000000000000000000000000000000000000000000000000000000000000005
```
____________
Similar for address of params:
---
```
def left_padded_address_of_param(param_index: int, width: int=64):
    bits_offset = 32
    bits_per_param = 64
    bits_for_the_param = bits_offset + bits_per_param * param_index
    return make_number_hex_left_padded(bits_for_the_param, width)
 ```
for this:
```
left_padded_address_of_param(1)
left_padded_address_of_param(2)
left_padded_address_of_param(3)
```
would generate:
```
'0000000000000000000000000000000000000000000000000000000000000060'
'00000000000000000000000000000000000000000000000000000000000000a0'
'00000000000000000000000000000000000000000000000000000000000000e0'
```